protocols/relay/src/behaviour/handler.rs

```diff
@@ -380,11 +382,18 @@ pub struct Handler {
     alive_lend_out_substreams: FuturesUnordered<oneshot::Receiver<()>>,
     /// Futures relaying data for circuit between two peers.
     circuits: Futures<(CircuitId, PeerId, Result<(), std::io::Error>)>,
+
+    stop_requested_streams: VecDeque<outbound_stop::StopCommand>,
+    protocol_futs: futures_bounded::WorkerFutures<(), CHEvent>,
```

Comment:
I don't find this property name intuitive. That said I can not think of a better name. I am fine merging with the current name.

Reply:
How about `workers` or `worker_streams`?